src/containers/Contact/index.js

```diff
@@ -111,8 +111,8 @@ class Contact extends Component {
   renderSubmitSuccess = () => (
```

Comment:
This and below, just making lines of relatively equal length, easier to edit if need be.